omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -700,14 +708,18 @@ def build_figure(self):
         # Add thumbnails and links page
         self.add_info_page(panels_json)
 
-        # Saves the completed  figure file
+        # Saves the completed figure file
         self.save_figure()
 
         # PDF will get created in this group
         if group_id is None:
             group_id = self.conn.getEventContext().groupId
         self.conn.SERVICE_OPTS.setOmeroGroup(group_id)
 
+        file_ann = self.create_file_annotation(image_ids)
```

Comment:
no real to assign file_ann can do ``return self.create_file_annotation(image_ids)``